Cookbook/Current.md

```diff
@@ -0,0 +1,22 @@
+# How to use `Current` in Babylon iOS Project
+
```

Comment:
I would explain what Current is and what its purpose is first.

Reply:
https://github.com/Babylonpartners/ios-playbook/blob/master/Cookbook/Proposals/ControlTheWorld.md

Reply:
also this is a TBD.

Reply:
Yes, but that is a proposal. This is the documentation for it -- I think it should serve as an independent and self-sufficient document explaining what Current is, what it does, how to use it, future directions, etc.

Reply:
> also this is a TBD contents.
Then maybe we should have this as WIP?

Reply:
I agree. But it's the beginning. It'll grow.

Reply:
I feel like at the current state, and as it is public, this document is a bit too barebones to be merged.

Reply:
I see no problem: 
- there's no link to this document;
- it's a documentation and there's nothing wrong on the content and publishing an in progress document;
- life is asynchronous 😛

Reply:
@diegopetrucci We need to speed up documentations.

Reply:
But you still have to go through PR approval whenever you want to make a change, so why not beefing it up a bit more for the first draft?

Reply:
the first idea was to create *just* the title.. and I did a lot more.

Reply:
> I would explain what Current is and what its purpose is first.

there's a reference to the proposal at the bottom.